### DISCUSSION TITLE
Stars Wars no Metrô

### DISCUSSION THREAD

**Discussion by vish-blog** [vish-blog/comments#451] [Vídeos]
Post de referência: https://www.vish.blog.br/videos/star-wars-no-metro/

**Comment by one-eyed-willy** [comment]
"Rebelião Galática para Leigos"

**Comment by angellisy** [comment]
> [Willy Caolho](</perfil/?user=one-eyed-willy>) disse [segunda-feira, 4 de abril de 2022](<https://www.vish.blog.br/videos/star-wars-no-metro/#comment_1087448772>):

> "Rebelião Galática para Leigos"

"Edição da Princesa" :D
